.travis.yml

```diff
@@ -10,13 +10,15 @@ python:
   - "pypy"
 
 env:
-  - AWS_SECRET_ACCESS_KEY=fake_key AWS_ACCESS_KEY_ID=fake_id
+  - AWS_SECRET_ACCESS_KEY=fake_key AWS_ACCESS_KEY_ID=fake_id UPDATE_BOTOCORE=0
```

Comment:
Would you mind pinning a newer `botocore` (e.g 1.11.4) in `requirements-dev.txt` instead of the out-of-band update? This will help avoid random test failures

Reply:
As you wish, it was to be sure to catch in advance if a new botocore version make another breaking change

Reply:
Yep, understand the motivation there. I think in practice we'll know about breaking changes about as quickly anyway and not floating here would avoid disruption to other ongoing development. Long-term I certainly agree that it'd be best to not be so dependent on internal behavior :)